src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -533,17 +538,22 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         });
     };
 
+    onTaskModalOpen = () => {
+        this.store.setItem(SIDEBAR_FORCE_KEY, SIDEBAR_FORCE_VALUE_OPEN);
+    };
+
     refresh(): void {
         this.fetchFeedItems(true);
     }
 
     renderAddTaskButton = () => {
         const { isDisabled } = this.props;
         const { approverSelectorContacts } = this.state;
-        const { getApproverWithQuery, getAvatarUrl, createTask, onTaskModalClose } = this;
+        const { getApproverWithQuery, getAvatarUrl, createTask, onTaskModalClose, onTaskModalOpen } = this;
         const props = {
             isDisabled,
             onTaskModalClose,
```

Comment:
Should there be associated callback to set the cache value as needed for the close case?